service-front/module/Application/view/application/pages/how_will_the_donor_confirm.twig

```diff
@@ -5,220 +5,263 @@
-                                    <li>UK passport (up to 18 months expired)</li>
-                                    <li>UK photocard driving licence</li>
-                                    <li class="govuk-!-padding-bottom-4">UK biometric residence permit</li>
-                                    <li>International passport</li>
-                                    <li>EU/EEA photocard driving licence</li>
-                                    <li>EU/EEA national identity card</li>
-                                </ul>
-                            </div>
-                        </details>
+                                    </summary>
+                                    <div class="govuk-details__text">
+                                        <ul class="govuk-list govuk-list--bullet govuk-hint">
+                                            <li>UK passport (up to 18 months expired)</li>
+                                            <li>UK photocard driving licence</li>
+                                            <li class="govuk-!-padding-bottom-4">UK biometric residence permit</li>
+                                            <li>International passport</li>
+                                            <li>EU/EEA photocard driving licence</li>
+                                            <li>EU/EEA national identity card</li>
+                                        </ul>
+                                    </div>
+                                </details>
 
-                    </label>
-                </div>
-            {% else %}
-                <div class="govuk-warning-text">
-                    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-                    <strong class="govuk-warning-text__text">
-                        <span class="govuk-visually-hidden">Warning</span>
-                        Post Office verification is not presently available
-                    </strong>
+                            </label>
+                        </div>
+                    {% else %}
+                        <div class="govuk-warning-text">
+                            <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                            <strong class="govuk-warning-text__text">
+                                <span class="govuk-visually-hidden">Warning</span>
+                                Post Office verification is not presently available
+                            </strong>
+                        </div>
+                    {% endif %}
                 </div>
-            {% endif %}
-        </div>
-        <details class="govuk-details">
-            <summary class="govuk-details__summary">
+                <details class="govuk-details">
+                    <summary class="govuk-details__summary">
             <span class="govuk-details__summary-text">
                 Other methods
```

Comment:
in the figma it looks like this has been changed to a heading with some slightly different wording for the options.

Reply:
Well spotted. I didn't notice that. Not very good at spot the difference apparently.

Reply:
Done

Reply:
Hey @mohammedrmoj, this still doesn't quite match the figma.

![figma](https://github.com/user-attachments/assets/1f447982-6e64-4905-b210-5e1738eda8a2)
vs
![actual](https://github.com/user-attachments/assets/f2eb23c9-94a6-40bd-aebf-a47a3eccc849)